### PR TITLE
python3Packages.python-telegram: fix build

### PR DESCRIPTION
**Pull request by sikmir**
## Description of changes
Fix https://hydra.nixos.org/build/239153404

## Things done
- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [x] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/telegram-text/default.nix [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  nativeBuildInputs = [
+    poetry-core
+  ];
+
+  nativeCheckInputs = [
+    pytestCheckHook
+  ];
+
+  meta = with lib; {
+    description = "Python markup module for Telegram messenger";
```
Comment (mweinelt) [inline pkgs/development/python-modules/telegram-text/default.nix:32]:
```suggestion
		changelog = "https://github.com/SKY-ALIN/telegram-text/blob/v${version}/CHANGELOG.md";
    description = "Python markup module for Telegram messenger";
    downloadPage = "https://github.com/SKY-ALIN/telegram-text";
```

**Comment by mweinelt** [comment]
Result of `nixpkgs-review pr 266730` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>10 packages built:</summary>
  <ul>
    <li>python310Packages.python-telegram</li>
    <li>python310Packages.python-telegram.dist</li>
    <li>python310Packages.telegram-text</li>
    <li>python310Packages.telegram-text.dist</li>
    <li>python311Packages.python-telegram</li>
    <li>python311Packages.python-telegram.dist</li>
    <li>python311Packages.telegram-text</li>
    <li>python311Packages.telegram-text.dist</li>
    <li>tg</li>
    <li>tg.dist</li>
  </ul>
</details>

File: pkgs/development/python-modules/telegram-text/default.nix (hunk 1 above)
Comment (mweinelt) [reply pkgs/development/python-modules/telegram-text/default.nix:32]: Please the `downloadPage` as well, I want to be able to reach the source repository directly.

**Comment by sikmir** [comment]
Result of `nixpkgs-review pr 266730` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>10 packages built:</summary>
  <ul>
    <li>python310Packages.python-telegram</li>
    <li>python310Packages.python-telegram.dist</li>
    <li>python310Packages.telegram-text</li>
    <li>python310Packages.telegram-text.dist</li>
    <li>python311Packages.python-telegram</li>
    <li>python311Packages.python-telegram.dist</li>
    <li>python311Packages.telegram-text</li>
    <li>python311Packages.telegram-text.dist</li>
    <li>tg</li>
    <li>tg.dist</li>
  </ul>
</details>

**Pull request merged**